hana/enable_secondary.sls

```diff
@@ -18,8 +30,7 @@
     - operation_mode: {{ node.secondary.operation_mode }}
     - timeout: {{ node.secondary.primary_timeout|default(100) }}
     - interval: {{ node.secondary.interval|default(10) }}
-    - primary_pass: {{ primary_pass }}
+    - primary_pass: {{ password.primary }}
```

Comment:
Hi @arbulu89 does the password variable need to be defined as dict here, I was wondering if it works as such

Reply:
Yes, this is the way. Actually, I was forced to use a dictionary, otherwise the variables defined in a for loop only have local scope, so you cannot use them outside. That's why the dictionaries are used here.